### PR TITLE
chore: log error message

### PR DESCRIPTION
**Pull request by ittus**
## Description

Print log error message.
Sometime it's difficult to get information from 
 `logging.error("", exc_info=True)`, so printing `logging.error(e)` will help a lot when debugging code.

## Type of change

- Chore

### CODE REVIEW

**Comment by msaroufim** [comment]
Hi @ittus do you mind showing us an example of an error logged before and after your change?

**Review by msaroufim — APPROVED** [review]

**Comment by ittus** [comment]
Hi @msaroufim 
Please check below log
You can see the additional log line
```
2022-02-28 07:23:40,970 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - In 'config.yaml': Could not find 'test/default'
```

Before
```ini
2022-02-28 07:18:06,613 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - ***** /tmp/models/9e2e3e2b99804195be63bdec4a0498cc
2022-02-28 07:18:06,614 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - Backend worker process died.
2022-02-28 07:18:06,614 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - Traceback (most recent call last):
2022-02-28 07:18:06,616 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_service_worker.py", line 176, in <module>
2022-02-28 07:18:06,622 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     worker.run_server()
2022-02-28 07:18:06,626 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_service_worker.py", line 148, in run_server
2022-02-28 07:18:06,628 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     self.handle_connection(cl_socket)
2022-02-28 07:18:06,628 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_service_worker.py", line 112, in handle_connection
2022-02-28 07:18:06,631 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     service, result, code = self.load_model(msg)
2022-02-28 07:18:06,632 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_service_worker.py", line 85, in load_model
2022-02-28 07:18:06,633 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     service = model_loader.load(model_name, model_dir, handler, gpu, batch_size)
2022-02-28 07:18:06,638 [INFO ] epollEventLoopGroup-5-2 org.pytorch.serve.wlm.WorkerThread - 9001 Worker disconnected. WORKER_STARTED
2022-02-28 07:18:06,639 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_loader.py", line 84, in load
2022-02-28 07:18:06,640 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     module = importlib.import_module(module_name)
2022-02-28 07:18:06,642 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
2022-02-28 07:18:06,644 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     return _bootstrap._gcd_import(name[level:], package, level)
2022-02-28 07:18:06,646 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
2022-02-28 07:18:06,646 [DEBUG] W-9001-test_1.0 org.pytorch.serve.wlm.WorkerThread - System state is : WORKER_STARTED
2022-02-28 07:18:06,648 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
2022-02-28 07:18:06,654 [INFO ] W-9000-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - ***** /tmp/models/9e2e3e2b99804195be63bdec4a0498cc
2022-02-28 07:18:06,664 [INFO ] epollEventLoopGroup-5-1 org.pytorch.serve.wlm.WorkerThread - 9000 Worker disconnected. WORKER_STARTED
```

After
```
2022-02-28 07:23:40,968 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - ***** /tmp/models/73fcca32da8d4d15a574cdd05768d53e
2022-02-28 07:23:40,970 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - In 'config.yaml': Could not find 'test/default'
2022-02-28 07:23:40,971 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -
2022-02-28 07:23:40,973 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - Config search path:
2022-02-28 07:23:40,976 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - 	provider=hydra, path=pkg://hydra.conf
2022-02-28 07:23:40,977 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - 	provider=schema, path=structured://
2022-02-28 07:23:40,980 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - Backend worker process died.
2022-02-28 07:23:40,981 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle - Traceback (most recent call last):
2022-02-28 07:23:40,981 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_service_worker.py", line 176, in <module>
2022-02-28 07:23:40,982 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     worker.run_server()
2022-02-28 07:23:40,982 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_service_worker.py", line 148, in run_server
2022-02-28 07:23:40,983 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     self.handle_connection(cl_socket)
2022-02-28 07:23:40,983 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -   File "/usr/local/lib/python3.8/dist-packages/ts/model_service_worker.py", line 112, in handle_connection
2022-02-28 07:23:40,983 [INFO ] W-9001-test_1.0-stdout org.pytorch.serve.wlm.WorkerLifeCycle -     service, result, code = self.load_model(msg)
2022-02-28 07:23:40,986 [INFO ] epollEventLoopGroup-5-1 org.pytorch.serve.wlm.WorkerThread - 9001 Worker disconnected. WORKER_STARTED
```

**Comment by msaroufim** [comment]
This seems substantially better - @maaquib @lxning for second stamp

File: ts/model_service_worker.py [hunk 1]
```diff
@@ -193,7 +193,8 @@ def run_server(self):
 
     except socket.timeout:
         logging.error("Backend worker did not receive connection in: %d", SOCKET_ACCEPT_TIMEOUT)
-    except Exception:  # pylint: disable=broad-except
+    except Exception as e:  # pylint: disable=broad-except
+        logging.error(e)
```
Comment (lxning) [inline ts/model_service_worker.py:197]: original line 197 already sets exc_info=True, so it is not necessary to add this.

Comment (ittus) [reply ts/model_service_worker.py:197]: Hi @lxning,
Could you please explain a bit more? Do you mean calling `logger.error(e)` is unnecessary?

To explain a bit more, sometime I can see the error message from `logging.error("Backend worker process died.", exc_info=True)`, but sometimes can't not.
It seems that after the worker is restarted (`Worker disconnected. WORKER_STARTED`), error message is truncated somehow.
So normally when debugging, I edit this file to add `logger.error(e)` to read error message easier.

**Comment by msaroufim** [comment]
Since this has been open for a while I believe it's best to close it. I believe we should take a deeper look at our overall logging strategy and make sure we surface the right error messages to our users but should do this as part of a more concerted effort

**Pull request closed without merging**